edu-services/cm-service/cm-impl/hibernate-impl/hibernate/src/hibernate/org/sakaiproject/coursemanagement/impl/EnrollmentCmImpl.hbm.xml

```diff
@@ -25,6 +25,7 @@
 		<property name="credits" column="CREDITS" type="string" not-null="true" />
 		<property name="gradingScheme" column="GRADING_SCHEME" type="string" not-null="true" />
 		<property name="dropped" column="DROPPED" type="boolean" />
+    <property name="dropDate" type="date" column="DROP_DATE" />
```

Comment:
can you fix this tab vs space issue?  In vim, ":set list" will show you tab characters.